### DISCUSSION TITLE
KERI Implementer's Call 05/23/2024

### DISCUSSION THREAD

**Discussion by pfeairheller** [WebOfTrust/keri#85] [Meetings]
The Primary contact (or backup) for each repo please comment on this discussion prior to the meeting date listed in the title with your update for the week...

KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
WOT-Terms - @henkvancann

**Comment by pfeairheller** [comment]
SigPy:

- No updates

**Comment by kordwarshuis** [comment]
Kerific:

- No updates

**Comment by 2byrds** [comment]
KERIA:
https://github.com/WebOfTrust/keria/issues/230 is wrapping up.
@pfeairheller updated KERIpy to run parsing locally and provide a delegation escrow processor https://github.com/WebOfTrust/keripy/pull/786/files
@pfeairheller update KERIA agents to leverage the newly available delegation escrow https://github.com/WebOfTrust/keria/pull/246
@2byrds made a complimentary PR to allow the user to approve the delegation after the interaction seal has been added to the KEL and successfully ran the test that @lenkan created as part of issue 230 https://github.com/WebOfTrust/keria/pull/247/files

**Comment by henkvancann** [comment]
Kerisse (WOT-terms):
Request [issue 147](https://github.com/WebOfTrust/WOT-terms/issues/147): The KERI/ACDC/CESR specifications in spec-up have just one source file: `spec.md`. This makes it hard to make the term definitions widely available. Could somebody at least cut it in three parts, having the terms-definitions in a separate .md file, called `terms_and_definitions.md` in each spec.

**Reply by henkvancann** [reply]
None of the administrators of the repos at hand happened to be present. @SmithSamuelM, @m00sey, @pfeairheller : would you accept a PR to logically split the spec.md up in at least three sources files per repo, once we mitigated the objection and offer edit buttons per section on the frontend side? Spec-up combines the sections into one spec using specs.json.

**Reply by pfeairheller** [reply]
Yes, I am fine with that approach

**Reply by henkvancann** [reply]
Done: PRs of the splitted spec.md with clear references to the sub file names for KERI, ACDC and CESR
https://github.com/trustoverip/tswg-keri-specification/pull/179
https://github.com/trustoverip/tswg-cesr-specification/pull/100
https://github.com/trustoverip/tswg-acdc-specification/pull/104
The KERI spec PR looks a bit messy, but it's not. The reason I wasn't able to branch off this activity was that spec-up / github.io wouldn't generate my production site on my user account on the new branch. After looking into it for 2 hrs I chose the practical way: if you accept the PRs and merge them,  I will bring in again the older (already discussed and accepted) changes I proposed earlier, but hadn't come to a merge yet

**Comment by m00sey** [comment]
keripy updates for fixing delegation on main:

https://github.com/WebOfTrust/keripy/pull/784
https://github.com/WebOfTrust/keripy/pull/786

**Comment by kentbull** [comment]
## SigTS
### Merged PRs
- [fix: remove development from workflow triggers](https://github.com/WebOfTrust/signify-ts/pull/262)
  - good for publishing an NPM package
- [fix: do not include src files in npm package](https://github.com/WebOfTrust/signify-ts/pull/261)
- [signedFetch more generic/configurable aligning it with fetch api](https://github.com/WebOfTrust/signify-ts/pull/259)
### New PRs
- [fix: export hab state types from main export](https://github.com/WebOfTrust/signify-ts/pull/263/files)
- [fix: delete contact by prefix](https://github.com/WebOfTrust/signify-ts/pull/264)

Other than this there has been some dependent work in KERIA and KERIpy to fix delegation.

## CESRide
Nothing new the last week

**Comment by edeykholt** [comment]
Discussion topics?
1) Signify-ts queries for SignifyClient and the managed identifiers.
2) Discussion of user experience topics (from ACME-V 4/4 call)  https://docs.google.com/document/d/14OxmKHltU3PW79uZsk7WP9HsMUbXqpcd3VAqkArWqw4/edit#heading=h.q6lfh4clrcuk

**Comment by pSchlarb** [comment]
Just bringing https://github.com/WebOfTrust/keripy/issues/775 to attention, not sure if here is the right place to do so.
